src/main/kotlin/by/architecture_map/belarus/service/SearchService.kt

```diff
@@ -0,0 +1,14 @@
+package by.architecture_map.belarus.service
+
+import by.architecture_map.belarus.entity.Construction
+
+interface SearchService {
+
+    fun constructionSearch(
+        architecturalStyleId: String?,
```

Comment:
if the amount of parameters is too big, it is better to use DTO
but up too 5 parameters is acceptable